libs/ui-toolkit/src/components/toast/toast.tsx

```diff
@@ -118,26 +206,83 @@ export const Toast = ({
           type="button"
           data-testid="toast-close"
           onClick={closeToast}
-          className="absolute p-2 top-0 right-0"
+          className="absolute p-[8px] top-[3px] right-[3px] z-20"
         >
-          <Icon name="cross" size={3} className="!block dark:text-white" />
+          <Icon
+            name="cross"
+            size={3}
+            className="!block dark:text-white !w-[11px] !h-[11px]"
+          />
         </button>
         <div
-          className={classNames(getToastAccent(intent), 'p-2 pt-3 text-center')}
+          data-testid="toast-accent"
+          className={classNames(
+            {
+              // gray
+              'bg-vega-light-200 dark:bg-vega-dark-200 text-vega-light-400 dark:text-vega-dark-100':
+                intent === Intent.None,
+              // blue
+              'bg-vega-blue-500 text-vega-blue-600': intent === Intent.Primary,
+              // green
+              'bg-vega-green-500 text-vega-green-600':
+                intent === Intent.Success,
+              // orange
+              'bg-vega-orange-500 text-vega-orange-600':
+                intent === Intent.Warning,
+              // pink
+              'bg-vega-pink-500 text-vega-pink-600': intent === Intent.Danger,
+            },
+            'w-8 p-[9px]',
+            'flex justify-center'
+          )}
         >
           {loader ? (
-            <div className="w-4 h-4">
+            <div className="w-[15px] h-[15px]">
               <Loader size="small" forceTheme="dark" />
             </div>
           ) : (
-            <Icon name={toastIconMapping[intent]} size={4} className="!block" />
+            <Icon
+              name={toastIconMapping[intent]}
+              size={4}
+              className="!block !w-[14px] !h-[14px]"
+            />
           )}
         </div>
         <div
-          className="flex-1 p-2 pr-6 text-sm overflow-auto dark:bg-black dark:text-white"
+          className={classNames(
+            'relative',
+            'overflow-auto flex-1 p-4 pr-[40px] [&>p]:mb-[2.5px]',
+            // toast heading
+            '[&>h3]:text-[14px] [&>h3]:leading-[13px] [&>h3]:uppercase [&>h3]:mb-[8px]'
```

Comment:
suggestion (non blocking): I don't think we should add these styles. If you were to create a different toast its not obvious that you should use an H3. Also the toast component shouldn't dictate what content and elements are required within it. I think we should one of the following

I would apply these styles to directly to the h3 element, maybe make a ToastHeading component to keep things DRY.

Reply:
Additionally, I'd like to try and avoid using specific px values where possible so we can keep consistency, yes it might not be pixel perfect compared to the designs but there is more value in the long run. The more we deviate from the TW config the more messy it will start to look over time

I just changed
`'[&>h3]:text-[14px] [&>h3]:leading-[13px] [&>h3]:uppercase [&>h3]:mb-[8px]'`
to
`'[&>h3]:text-sm [&>h3]:uppercase [&>h3]:mb-1'`

and the end result is almost identical